src/controlCenter/widgets/volume/sinkInputRow.vala

```diff
@@ -3,14 +3,18 @@ namespace SwayNotificationCenter.Widgets {
 
         Gtk.Box container;
         Gtk.Image icon = new Gtk.Image ();
+        Gtk.Label label = new Gtk.Label ("");
         Gtk.Scale scale = new Gtk.Scale.with_range (Gtk.Orientation.HORIZONTAL, 0, 100, 1);
 
         public unowned PulseSinkInput sink_input;
 
         private unowned PulseDaemon client;
 
-        public SinkInputRow (PulseSinkInput sink_input, PulseDaemon client, int icon_size) {
+        private bool show_per_app_label;
+
+        public SinkInputRow (PulseSinkInput sink_input, PulseDaemon client, int icon_size, bool show_per_app_label) {
```

Comment:
Also, this line is a bit long. Mind making it take up multiple lines instead? :)